### DISCUSSION TITLE
Icons or font-size for lights?

### DISCUSSION THREAD

**Discussion by ceejii** [cgiesche/streamdeck-homeassistant#219] [Help and Support]
So I added 7 lights and switches. They all have the same on/off icon as expected. The font-size on the buttons is also pretty small. I need some way of making it clearer which switch is for witch light/switch so I don't have to press randomly. Is there any way to make it clearer which button is for which entity in Home Assistant? Maybe get the entity icon from HA, maybe enable larger font size?

(I can make out the text if I squint and look for 5-10 seconds, but it's really slow. I don't have perfect vision for short range due to choosing to not use bifocals and my eyes getting "old".)

**Comment by cgiesche** [comment]
Hi ceejii,

currently, the only option is to use the `title` or `custom title` field. You can change font size and position for this field. This is a "dirty" workaround:

![image](https://github.com/cgiesche/streamdeck-homeassistant/assets/5123180/2fb04ae7-9365-4d06-a09d-40b69a40fbbd)

1. Change font size and position here
2. Change title as needed
3. Change custom labels to ` ` (space) to suppress any overlapping texts.

A more long term solution:
I am planning to allow to disable the icons that i made for some entities (like switches and lights) and by that allow to use the icons provided 
and configured by home-assistant.

**Comment by Molecule5537** [comment]
Unsure if this is the right thread, but I would love to be able to change the font size on the custom labels.
